src/docbuilder/rustwide_builder.rs

```diff
@@ -923,6 +944,12 @@ mod tests {
                 }
             }
 
+            assert!(matches!(*env.cdn(), CdnBackend::Dummy(_)));
+            if let CdnBackend::Dummy(ref invalidation_requests) = *env.cdn() {
```

Comment:
```suggestion
            let CdnBackend::Dummy(ref invalidation_requests) = *env.cdn() else { panic!("wrong cdn kind") };
```

Reply:
Actually no, that's only just made it to beta, another 6 weeks till it's stable.